components/brave_wallet_ui/components/shared/datatable/style.ts

```diff
@@ -0,0 +1,144 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { SortOrder } from '../../../constants/types'
+import styled, { css } from 'styled-components'
+import { Cell, Row } from './index'
+
+import ArrowDownIcon from '../../../assets/svg-icons/arrow-down-fill.svg'
+import ArrowUpIcon from '../../../assets/svg-icons/arrow-up-fill.svg'
+
+export interface StyleProps {
+  customStyle?: {[key: string]: string}
```

Comment:
All styled components already come with a built in way to override styles.
![Screen Shot 2022-04-25 at 3 14 55 PM](https://user-images.githubusercontent.com/30185185/165176309-befc2b58-f4a5-448d-8184-ffacd25a9d21.png)